spacetime/tests/test_linear_scalar.py

```diff
@@ -9,18 +9,17 @@
 
 
 class LinearScalarSolverTC(unittest.TestCase):
-
     @staticmethod
     def _build_solver(resolution):
 
         # Build grid.
-        xcrd = np.arange(resolution+1) / resolution
+        xcrd = np.arange(resolution + 1) / resolution
```

Comment:
This change has nothing to do with your addition of error analysis.  I would hope that the change you make to existing files is only about the planned feature / fix.  It is not needed to change the coding style of existing files.

Please revert this and other style changes alike.

Reply:
I'd like to move this PR forward.  Would it be good to have the unnecessary changes reverted?

Reply:
I reverted these unnecessary changes in this commit https://github.com/j8xixo12/turgon/commit/38a4d66784497f65032bea2a63011f70f5f375ba

Reply:
There seems to be something wrong.  I cannot find the change in your branch.  I pulled your branch and the diff doesn't have the lines reverted: https://gist.github.com/yungyuc/abeb8d83ec7d2facaf8bd3c73af173ca, just like what GitHub is showing here in the review.

Can you take a look at what is going on?